src/applications/accredited-representative-portal/containers/SubmissionsPage.jsx

```diff
@@ -0,0 +1,192 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  useLoaderData,
+  useSearchParams,
+  useNavigation,
+} from 'react-router-dom';
+import {
+  VaLoadingIndicator,
+  VaBreadcrumbs,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { Toggler } from 'platform/utilities/feature-toggles';
+import { focusElement } from 'platform/utilities/ui';
+import api from '../utilities/api';
+import {
+  SUBMISSIONS_BC_LABEL,
+  submissionsBC,
+  SORT_DEFAULTS,
+} from '../utilities/submissions';
+import { SEARCH_PARAMS } from '../utilities/constants';
+import SortForm from '../components/SortForm';
+import Pagination from '../components/Pagination';
+import PaginationMeta from '../components/PaginationMeta';
+import SubmissionCard from '../components/SubmissionCard';
+
+const SearchResults = ({ submissions }) => {
+  if (!submissions || submissions.length === 0) {
+    return (
+      <p data-testid="submissions-table-fetcher-empty">
+        No form submissions found
+      </p>
+    );
+  }
+
+  return (
+    <ul
+      data-testid="submissions-card"
+      className="submissions__list"
+      sort-column={1}
+    >
+      {submissions.map((submission, index) => {
+        return <SubmissionCard submission={submission} key={index} />;
+      })}
+    </ul>
+  );
+};
+
+SearchResults.propTypes = {
+  submissions: PropTypes.arrayOf(
+    PropTypes.shape({
+      length: PropTypes.number,
+      map: PropTypes.func,
+    }),
+  ),
+};
```

Comment:
Should we move the `SearchResults` component into its own jsx file? (ditto for `POARequestSearchPage`)